doc/intro.rst

```diff
@@ -91,6 +91,8 @@ please use the DOI of the specific version that you have employed. You can
 find it by clicking on the DOI badge below.
 
 .. image:: https://zenodo.org/badge/doi/10.5281/zenodo.10252344.svg
+   :class: sd-bg-transparent
```

Comment:
I can't see any difference between this one and the one in `doc/citing.rst`, which doesn't have it. Could it be browser dependent?
According to https://sphinx-design.readthedocs.io/en/latest/css_classes.html#colors, this is suppose to change the background to transparent right? If so, could the effect be visible on the edges?

Reply:
Yes, without this there is a gray edge in dark mode. See also https://github.com/hyperspy/holospy/pull/27#discussion_r1545654623